cfme/automate/service_dialogs.py

```diff
@@ -1,84 +1,109 @@
-# -*- coding: utf-8 -*-
-import functools
-
-from navmazing import NavigateToSibling, NavigateToAttribute
-
 from cfme.fixtures import pytest_selenium as sel
-from cfme.web_ui import toolbar as tb
-from cfme.web_ui import AngularSelect, Form, Select, SplitTable, accordion,\
-    fill, flash, form_buttons, Table, Tree, Input, Region, BootstrapTreeview
-from utils import version
-from utils.appliance import Navigatable
-from utils.appliance.implementations.ui import navigator, navigate_to, CFMENavigateStep
-from utils.pretty import Pretty
+from widgetastic.widget import Text, Checkbox
+from widgetastic_patternfly import Button, Input, BootstrapSelect, Dropdown
+from widgetastic_manageiq import ManageIQTree, Table
+from navmazing import NavigateToAttribute, NavigateToSibling
 from utils.update import Updateable
+from utils.pretty import Pretty
+from utils.appliance import Navigatable
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+from utils import version
+
+
+from . import AutomateCustomizationView
+
+
+class LabelForm(AutomateCustomizationView):
+    title = Text('#explorer_title_text')
```

Comment:
Yes, this represents multitude of views in one, which is not good. We have View.include and subclassing, I guess that would simplify the code as well.